app/components/pages/Dashboard/index.js

```diff
@@ -29,6 +29,7 @@ const MobileOnlyStickyFooter = styled(StickyFooter)`
 `
 
 const CenterdSmallParagraph = styled(SmallParagraph)`
+  margin-top: 12px;
```

Comment:
As a general rule, we put `margin-bottom` on components wherever possible. We should record this somewhere. You weren't to know.

This is because combining `margin-bottom` on a higher-up component & `margin-bottom` on a lower-down component would result in margin collapse. So the unwritten rule of using only margin-bottom for vertical spacing avoids this scenario.

Could we pop `margin-bottom on the `DashboardPanel` instead?

Reply:
yes, sure!